client/client.ml

```diff
@@ -213,8 +213,13 @@ let log_cmd ~confdir ~conffile =
   (term, info)
 
 let cmds =
-  let confdir = XDGBaseDir.(default.config_home) in
-  let confdir = Fpath.v confdir in
+  let module App_id = struct
+      let qualifier = "org"
+      let organization = "ocurrent"
+      let application = "opam-repo-ci"
```

Comment:
Shouldn't this be directlty `opam-health-check` ? I see you're appending it after.